samples/client/petstore/typescript-node/api.ts

```diff
@@ -25,6 +25,20 @@ export class Dog extends Animal {
     "breed": string;
 }
 
+export class FormatTest {
```

Comment:
Im not sure why this is here. It is causing a compile error because ByteArray is undefined. I can't figure out what is generating this.